src/workspaces/common/state/useWorkspaces.ts

```diff
@@ -26,6 +27,7 @@ const defaultFieldsArgs: FieldsArg = [
  */
 export const useWorkspaces = (fieldsArg?: FieldsArg, stringAttributeMaxLength?: number): UseWorkspacesResult => {
   const signal = useCancellation();
+  const { reportError } = withErrorReporter(useNotificationsFromContext());
```

Comment:
because useWorkspaces hook is used by Environments component, it must switch to the context driven notifications so that external consumers of Environments (AoU) can provide their own notifications implementation through react context.